packages/echo/src/client.test.ts

```diff
@@ -344,9 +345,10 @@ describe('Echo Client', () => {
     it('should execute workflow successfully when action is execute and data is provided', async () => {
       await echo.workflow('test-workflow', async ({ step }) => {
         await step.email('send-email', async () => ({ body: 'Test Body', subject: 'Subject' }));
+        await step.delay('delay', async () => ({ type: 'regular', unit: 'seconds', amount: 1 }));
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** ‏ maybe pull out the configuration of the delay step to a const above so that it can be used in asserts below

Reply:
done 🔥